definitions/divorce/json/AuthorisationCaseState.json

```diff
@@ -1915,6 +1915,6 @@
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "AwaitingService",
     "UserRole": "caseworker-divorce-courtadmin",
-    "CRUD": "R"
+    "CRUD": "RU"
   }
 ]
```

Comment:
Just to make sure, the state 'ClarificationSubmitted" defined on the ticket does not exist yet right?

Reply:
@Dizrupto  yup, that state is not on prod yet